packages/commonwealth/client/scripts/views/pages/view_proposal/ViewProposalCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { capitalize } from 'lodash';
+import moment from 'moment';
+import React from 'react';
+import AddressInfo from '../../../models/AddressInfo';
+import app from '../../../state/index';
+import { CWContentPageCard } from '../../components/component_kit/cw_content_page';
+import { CWText } from '../../components/component_kit/cw_text';
+import { User } from '../../components/user/user';
+import { SnapshotThreadLink } from './proposal_header_links';
+
+export const ProposalInformationCard = ({ proposal, threads }) => {
```

Comment:
1. name of the component should be the same as the name of the file
2. Add interface with types